docs/MO_DG/Deep_Learning_Model_Optimizer_DevGuide.md

```diff
@@ -22,7 +22,7 @@
 
 Model Optimizer is a cross-platform command-line tool that facilitates the transition between the training and deployment environment, performs static model analysis, and adjusts deep learning models for optimal execution on end-point target devices.
```

Comment:
Can we rename Model Optimizer Developer Guide to Model Optimizer User Guide?

Also can we remove deprecated pages?

Reply:
Yes, we will definitely rename it.

Reply:
Done